frontend/packages/signless-transactions/src/components/create-session-modal/create-session-modal.tsx

```diff
@@ -87,39 +87,46 @@ function CreateSessionModal({ close }: Props) {
 
   return (
     <>
-      <Modal heading="Create Signless Session" close={close}>
+      <Modal heading="Enable Signless Session" close={close}>
         <ul className={styles.summary}>
-          <li>
+          <li className={styles['summary-item']}>
             <h4 className={styles.heading}>
               {storagePair ? 'Account from the storage:' : 'Randomly generated account:'}
             </h4>
-
+            <div className={styles.separator} />
             {pair && (
               <div className={styles.account}>
-                <Identicon value={pair.address} theme="polkadot" size={14} />
-                <span>{getVaraAddress(pair.address)}</span>
+                <span className={styles.value}>{shortenString(getVaraAddress(pair.address), 4)}</span>
```

Comment:
I don't think that shorten addresses via util function is a good practice in terms of accessibility. Let's stick to `text-overflow: ellipsis` style to shorten strings

Reply:
I thought ellipsis makes string to be shorten just from the end like abcd… But I haven’t seen ways how we can use it to make string in format abcd…abcd

Reply:
I don't think that there's any reason to stick to `abc...abc` trim over text overflow's `abc...` apart from design perspective. Accessibility should be the first priority.

Reply:
On all project designs I've seen we having addresses shown in format abcd...abcd. Should we discuss it with designers?

Reply:
Yes